tests/specifications/googlefonts_test.py

```diff
@@ -853,13 +853,13 @@ def test_check_085():
   some_bad_values = ["APACHE", "Apache", "Ufl", "Ofl", "Open Font License"]
 
   for good in good_licenses:
-    print ("Test PASS: ".format(good))
+    print (f"Test PASS: ")
     md.license = good
     status, message = list(check(md))[-1]
     assert status == PASS
 
   for bad in some_bad_values:
-    print ("Test FAIL: ".format(bad))
+    print (f"Test FAIL: ")
```

Comment:
`bad` got dropped here somehow; also above.

Reply:
Sorry about those, will fix these and report it as a bug to pyupgrade.

Reply:
Reverted, and reported in https://github.com/asottile/pyupgrade/issues/48.